### DISCUSSION TITLE
Open downloaded file

### DISCUSSION THREAD

**Discussion by parasit** [oae/kaizoku#82] [Q&A]
Hi,
I've just started using Kaizoku, I've used the console version of mangal before, and the first thing that caught my eye - why can't I open an already downloaded file from the browser? Something is not working in my installation or is it supposed to be like this?

I have a list of files, great, but to open them I have to log on to a remote computer (for me, kaizoku runs on a home linux "server"), copy them manually to my local machine and then open them, its not a big "problem" but a link to cbz/pdf would be enough to download/open them from browser level.

**Comment by Ambushfall** [comment]
Even though all of the mangal settings are included, I don't think a way to handle Open is added at all looking at the source code.

**Comment by vonProteus** [comment]
my solution is use kaizoku to download only
i have another webapp to read (komga)
which has 1 library in download folder of kaizoku

**Reply by parasit** [reply]
Maybe this will be solution, but still IMO will be easier just add links and open locally in my preferred application/device.
